web/packages/teleport/src/services/resources/makeUnifiedResource.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { UnifiedResource, UnifiedResourceKind } from '../agents';
+import makeApp from '../apps/makeApps';
+import { makeDatabase } from '../databases/makeDatabase';
+import { makeDesktop } from '../desktops/makeDesktop';
+import makeNode from '../nodes/makeNode';
+
+export function makeUnifiedResource(json: any): UnifiedResource {
+  json = json || {};
+
+  switch (json.kind as UnifiedResourceKind) {
+    case 'app':
+      return makeApp(json);
+    case 'db':
+      return makeDatabase(json);
+    case 'node':
+      return makeNode(json);
+    case 'windows_desktop':
+      return makeDesktop(json);
```

Comment:
no kubes?

Reply:
Wow, thanks for spotting this!

Reply:
So I added kubes here, but noticed that the API endpoint doesn't return kubes anyway. @avatus , I merged the latest backend changes here and I have one minikube in my configuration. It's visible through a standalone Kubernetes cluster list; however, it's missing from the API response. Could you please look into it? Is it just my setup?

Reply:
Not your setup. Found the getter on the backend was not pulling kubes correctly. Will get a fix in